services/create_ad.rb

```diff
@@ -0,0 +1,21 @@
+class CreateAd
```

Comment:
It would be better to put your services under _domain_ namespace like this: `Ads::CreateService`.